ledger/ledger-consensual/src/test/java/net/corda/v5/ledger/consensual/ConsensualLedgerServiceJavaApiTest.java

```diff
@@ -32,4 +39,58 @@ public void getTransactionBuilder() {
         Assertions.assertThat(consensualLedgerService.findSignedTransaction(fakeTxId)).isNull();
         Assertions.assertThat(consensualLedgerService.findLedgerTransaction(fakeTxId)).isNull();
     }
+
+    @Test
+    public void findSignedTransaction() {
+        SecureHash secureHash = new SecureHash("SHA-256", "123".getBytes());
+        ConsensualSignedTransaction consensualSignedTransaction = mock(ConsensualSignedTransaction.class);
+        when(consensualLedgerService.findSignedTransaction(secureHash)).thenReturn(consensualSignedTransaction);
+
+        ConsensualSignedTransaction result = consensualLedgerService.findSignedTransaction(secureHash);
+
+        Assertions.assertThat(result).isNotNull();
+        Assertions.assertThat(result).isEqualTo(consensualSignedTransaction);
+        verify(consensualLedgerService, times(1)).findSignedTransaction(secureHash);
+    }
+
+    @Test
+    public void findLedgerTransaction() {
+        SecureHash secureHash = new SecureHash("SHA-256", "123".getBytes());
+        ConsensualLedgerTransaction consensualLedgerTransaction = mock(ConsensualLedgerTransaction.class);
+        when(consensualLedgerService.findLedgerTransaction(secureHash)).thenReturn(consensualLedgerTransaction);
+
+        ConsensualLedgerTransaction result = consensualLedgerService.findLedgerTransaction(secureHash);
+
+        Assertions.assertThat(result).isNotNull();
+        Assertions.assertThat(result).isEqualTo(consensualLedgerTransaction);
+        verify(consensualLedgerService, times(1)).findLedgerTransaction(secureHash);
+    }
+
+    @Test
+    public void finality() {
```

Comment:
I feel like `finality` is the wrong name here. Typically function names should be verbs. As `finality` is a noun, perhaps `finalize` would be better.

@lankydan @mnesbit thoughts?

Reply:
Personally, I don't mind as long as `finality` and `receiveFinality` are clearly related.

Reply:
I admit if we expose the flow as a service it probably should be finalize as a method and Finality as a flow

Reply:
I have renamed `finality()` to `finalize()`. (I did not touch `receiveFinality()`... Let me know if you would prefer to change that too.)